azurerm/internal/services/servicebus/tests/data_source_servicebus_subscription_test.go

```diff
@@ -0,0 +1,68 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMServiceBusSubscription_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_servicebus_subscription", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMServiceBusSubscriptionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMServiceBusSubscription_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMServiceBusSubscriptionExists(data.ResourceName),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "max_delivery_count"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceAzureRMServiceBusSubscription_basic(data acceptance.TestData) string {
+	return fmt.Sprintf(testAccDataSourceAzureRMServiceBusSubscription_tfTemplate, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger, "")
+}
+
+const testAccDataSourceAzureRMServiceBusSubscription_tfTemplate = `
```

Comment:
(which means we could remove this constant)

Reply:
fixed !